eudemo/eudemo/reactor.py

```diff
@@ -238,12 +238,13 @@ def _read_json(file_path: str) -> Dict:
 if __name__ == "__main__":
     import time
 
-    reactor = EUDEMO("EUDEMO")
     params = make_parameter_frame(PARAMS_FILE, EUDEMOReactorParams)
     if params is None:
         raise ValueError("Params cannot be None")
     build_config = _read_json(os.path.join(CONFIG_DIR, "build_config.json"))
 
+    reactor = EUDEMO("EUDEMO", n_sectors=params.n_TF.value)
```

Comment:
This is a shame that we have to pass this into the top-level of the reactor. Feels extremely hacky, but I can't think of a better way... This is actually an API breaking change, as `n_sectors` is a required argument. 

Is there a way to, by default, assume that we've built all the sectors in xyz, but have this as an optional argument that tells the reactor that we've only built 1 sector out of `N`? Or does that not make sense for the way we've done things?

Also, there's no docstring for `n_sectors` in the `Reactor` class.